attest/verifier/types/src/verification.rs

```diff
@@ -151,6 +154,63 @@ impl Message for VerificationSignature {
     }
 }
 
+/// Structure for holding the contents of the Enclave's Report Data.
+/// The Enclave's ReportData member contains a SHA256 hash of this structure's
+/// contents.
+#[derive(Debug, Clone)]
+pub struct EnclaveReportDataContents {
+    nonce: QuoteNonce,
+    key: X25519Public,
+    custom_identity: [u8; 32],
+}
+
+impl EnclaveReportDataContents {
```

Comment:
I should mention this is in this module because the newer DCAP version of the `VerificationReport` will be coming into this file soonish and that new type will eventually contain an instance of this type.